drivers/sqlboiler-psql/driver/psql.go

```diff
@@ -90,6 +91,11 @@ func (p *PostgresDriver) Assemble(config drivers.Config) (dbinfo *drivers.DBInfo
 		}
 	}()
 
+	p.version, err = p.getVersion()
+	if err != nil {
+		return nil, errors.Wrap(err, "sqlboiler-psql failed to get database version")
```

Comment:
Will this succeed 100% on all postgresql versions? If not, it should not return an error and it should fall back to the old behavior.

Reply:
I can find this option from [postgresql documentation since 8.2](https://www.postgresql.org/docs/8.2/runtime-config-preset.html). So technically not all versions will succeed. But all officially supported versions (>= 9.5) are supported. Do you think if it is fine to assume it just works?

If not, about the fallback, there can be different ways to achieve the old behaviour, e.g.
```go
// 1. introduce one more flag about if getVersion() succeeds:
// whenever we need to distinguish the version for a sql statement
if p.getVersionSucceeds && p.version >= 120000 {
  sql = "new behaviour"
} else {
  sql = "old behaviour"
}

// 2. just set p.version = 0 if getVersion() fails:
if p.version >= 120000 {
  sql = "new behaviour"
} else {
  sql = "old behaviour"
}
```
Which one do you prefer? Do you have any idea?

Reply:
As long as it works on officially supported its fine by me.